### PR TITLE
Changelog update for stable branch.

### PR DESCRIPTION
**Pull request by itsjustkevin**
Updates changelog on stable.  Updates here will be merged to master.

### CODE REVIEW

**Comment by itsjustkevin** [comment]
@jmagman and @reidbaker could you check the changelog entries for 3.24.5 to make sure they make sense?

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,8 +24,22 @@ docs/releases/Hotfix-Documentation-Best-Practices.md
...
 -->
+## Flutter 3.27 Changes
+
+### [3.27.0](https://github.com/flutter/flutter/releases/tag/3.27.0)
+Initial stable release.
+
 ## Flutter 3.24 Changes
 
+### [3.24.5](https://github.com/flutter/flutter/releases/tag/3.24.5)
+- [flutter/158125](https://github.com/flutter/flutter/pull/158125) - [Tool] Fixed a tool issue causing failures when running `flutter build ios-framework --xcframework`.
```
Comment (jmagman) [inline CHANGELOG.md:35]:
```suggestion
- [flutter/158125](https://github.com/flutter/flutter/pull/158125) - [iOS] Fixed a tool issue causing failures when `flutter build ios-framework --xcframework` copies Flutter debug symbols.
```

**Review by jmagman — APPROVED** [review]
LGTM with nit

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,8 +24,22 @@ docs/releases/Hotfix-Documentation-Best-Practices.md
...
+## Flutter 3.27 Changes
+
+### [3.27.0](https://github.com/flutter/flutter/releases/tag/3.27.0)
+Initial stable release.
+
 ## Flutter 3.24 Changes
 
+### [3.24.5](https://github.com/flutter/flutter/releases/tag/3.24.5)
+- [flutter/158125](https://github.com/flutter/flutter/pull/158125) - [iOS] Fixed a tool issue causing failures when `flutter build ios-framework --xcframework` copies Flutter debug symbols.
+- [flutter/56301](https://github.com/flutter/engine/pull/56301) - [Android] Fixes a crash on Android devices when the surface is released unexpectedly when using PlatformView's.
```
Comment (reidbaker) [inline CHANGELOG.md:36]: It is probably more helpful to link to https://github.com/flutter/engine/pull/55277

**Review by reidbaker — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (itsjustkevin) [reply CHANGELOG.md:36]: The already included pull request includes more information than the one you linked.  

We typically link the pull request or issue that matches the template.  Would you suggest replacing the link with the one you provided?

Comment (itsjustkevin) [reply CHANGELOG.md:36]: Merging this in the current state, will update further next week if necessary.

**Pull request merged**